pkg/nodejs/packagejson/parse_test.go

```diff
@@ -70,7 +73,19 @@ func TestParse(t *testing.T) {
 			}
 
 			require.NoError(t, err)
-			assert.Equal(t, v.want, got)
+			comparableKeys := []string{"Name", "Version", "License"}
+			inComparableKeys := []string{"ExternalReferences"}
+			for _, key := range comparableKeys {
+				expected := funk.Get(v.want, key)
+				actual := funk.Get(got, key)
+				assert.Equal(t, expected, actual)
+			}
+
+			for _, key := range inComparableKeys {
+				expected := funk.Get(v.want, key)
+				actual := funk.Get(got, key)
+				assert.ElementsMatch(t, expected, actual)
+			}
```

Comment:
We can sort external references

Reply:
We can, do you want to sort the refs here? or in each parser? @knqyf263

Reply:
@knqyf263 I added the sort here